### DISCUSSION TITLE
Xrdp for MS Windows?

### DISCUSSION THREAD

**Discussion by flyingosprey** [neutrinolabs/xrdp#3247] [Ideas]
Similar to https://github.com/neutrinolabs/xrdp/issues/2194 request, is it possbile to add MS Windows support?

I have this request because most home PC are "Windows Home" SKU, which does not have RDP serivce at all. I know there are some "hacking methods" to enable RDP service on Home SKU. But that breaks EULA, which I want to avoid.

Thanks.

**Comment by matt335672** [comment]
Hi @flyingosprey 

We discussed this in #2100, and I don't think anything has changed.
